wandb/sdk/internal/job_builder.py

```diff
@@ -135,7 +137,7 @@ def _handle_server_artifact(self, res: Dict, artifact: "ArtifactRecord") -> None
                 self._job_seq_id = res["artifactSequence"]["id"]
             except KeyError as e:
                 _logger.info(f"Malformed response from ArtifactSaver.save {e}")
-        if artifact.type == "code" and "id" in res:
+        if artifact.type == "code" and res is not None:
```

Comment:
in my original PR, I made it such that res is never None which is why I changed up the check. The revert changed the code path where it would be None and the check is fine